selfdrive/car/toyota/carstate.py

```diff
@@ -58,8 +57,8 @@ def update(self, cp, cp_cam):
     ret.steeringAngleDeg = cp.vl["STEER_ANGLE_SENSOR"]["STEER_ANGLE"] + cp.vl["STEER_ANGLE_SENSOR"]["STEER_FRACTION"]
     torque_sensor_angle_deg = cp.vl["STEER_TORQUE_SENSOR"]["STEER_ANGLE"]
 
-    # Some newer models have a more accurate angle measurement in the TORQUE_SENSOR message. Use if non-zero
-    if abs(torque_sensor_angle_deg) > 1e-3:
+    # Wait until an update has been seen to avoid learning an incorrect offset
+    if not bool(cp.vl["STEER_TORQUE_SENSOR"]["INITIALIZING"]):
```

Comment:
Not yet sure what the behavior on other models are with this bit, but TSS-P Prius and TSS2 RAV4 would work with without checking the angle signal for a non-zero value: https://imgur.com/a/WPJSmCP